keep-starknet-strange/unruggable.meme
# Add chart / explorer links

Resolves issue: https://github.com/keep-starknet-strange/unruggable.meme/issues/212

Steps to test:
- Run app locally.
- Now, go http://localhost:3000/token/0x06e2c03e31f6cda18484ed035c02a25d6075393e11da24c898337f736716acae.
- The above link is for a token which is launched so, you should be able to see the Dexscreener link.
- At the bottom you must see Starkscan link as well.


https://github.com/keep-starknet-strange/unruggable.meme/assets/45929094/904462de-4980-4fc8-af2a-f8a1a5b95591
